README.md

```diff
@@ -17,7 +17,7 @@ In addition we've provided a basic project scaffold for you to build from.
 You can use everything we've given you, although there's no requirement to use anything.
 ```bash
 # scaffold
-data 
+data
   - feed.json  # A sample feed data object
```

Comment:
Is this different at all in the PR?

Reply:
In terms of the filenames, and what's in them?

Reply:
Yeah there's a small change, updating now